packages/core/src/app/customer/CheckoutButtonContainer.tsx

```diff
@@ -54,8 +56,16 @@ const CheckoutButtonContainer: FunctionComponent<CheckoutButtonContainerProps &
         return null;
     }
 
+    const isPaypalCommerce = methodIds.includes('paypalcommerce');
```

Comment:
It might be better to add extra check for 'paypalcommercecredit' and 'paypalcommercevenmo' method ids too.

Reply:
Thanks. Done.